Chore/116 enhance company create preview styling

TerrellLinnell opened the pull request.

TerrellLinnell committed:
chore: Display company location above description

Moved render of location above render of description in preview component

#112

TerrellLinnell committed:
Merge pull request #135 from codexstanford/chore/display-company-location-above-company-description

chore: Display company location above description

TerrellLinnell committed:
chore: Add padding to logo preview

Added margin to logo in preview to give it some breathing room

#113

TerrellLinnell committed:
Merge pull request #136 from codexstanford/chore/add-margin/padding-to-company-logo

Chore/add margin/padding to company logo

TerrellLinnell committed:
chore: Display company links in company create preview

Rework company links display to be more flexible to use in multiple places, display links in company create preview

#114

TerrellLinnell committed:
Merge pull request #144 from codexstanford/chore/display-company-links-in-company-create-preview

chore: Display company links in company create preview

TerrellLinnell committed:
chore: Display company categories in company create preview

Reworked current intelligence component to accomodate preview and data from server, implemented component in preciew component

#115

TerrellLinnell committed:
chore: Remove left margin on target markets

Remove left margin on target markets under 480pc screen size

#115

TerrellLinnell committed:
Merge pull request #154 from codexstanford/chore/display-company-categories-in-company-create-preview

Chore/display company categories in company create preview

medelman17 commented:
Ohhh, I see what happened! You used the new ‘feature branch’ flow we used last sprint. I liked that. Let’s keep it up. Next time can you create a WIP PR for the branch just as soon as you create it? I don’t mind having tickets toggled to done when merged into the feature branch (as they should be)—I just need to know where to look to review. :-)

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: TerrellLinnell <notifications@github.com>
Sent: Monday, July 15, 2019 8:01:21 PM
To: codexstanford/techlist-frontend-web
Cc: Subscribed
Subject: [codexstanford/techlist-frontend-web] WIP: Chore/116 enhance company create preview styling (#155)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/codexstanford/techlist-frontend-web/pull/155

Commit Summary

  *   chore: Display company location above description
  *   Merge pull request #135 from codexstanford/chore/display-company-location-above-company-description
  *   chore: Add padding to logo preview
  *   Merge pull request #136 from codexstanford/chore/add-margin/padding-to-company-logo
  *   chore: Display company links in company create preview
  *   Merge pull request #144 from codexstanford/chore/display-company-links-in-company-create-preview
  *   chore: Display company categories in company create preview
  *   chore: Remove left margin on target markets
  *   Merge pull request #154 from codexstanford/chore/display-company-categories-in-company-create-preview

File Changes

  *   M src/apps/admin/features/company.create/components/basics.js<https://github.com/codexstanford/techlist-frontend-web/pull/155/files#diff-0> (3)
  *   M src/apps/admin/features/company.create/components/preview.js<https://github.com/codexstanford/techlist-frontend-web/pull/155/files#diff-1> (53)
  *   M src/templates/company.js<https://github.com/codexstanford/techlist-frontend-web/pull/155/files#diff-2> (10)
  *   M src/templates/company/contact.js<https://github.com/codexstanford/techlist-frontend-web/pull/155/files#diff-3> (154)
  *   M src/templates/company/helpers.js<https://github.com/codexstanford/techlist-frontend-web/pull/155/files#diff-4> (51)
  *   M src/templates/company/intelligence.js<https://github.com/codexstanford/techlist-frontend-web/pull/155/files#diff-5> (51)

Patch Links:

  *   https://github.com/codexstanford/techlist-frontend-web/pull/155.patch
  *   https://github.com/codexstanford/techlist-frontend-web/pull/155.diff

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/codexstanford/techlist-frontend-web/pull/155?email_source=notifications&email_token=ADQ3VLPMS4DWIAJ32WLFDJLP7UFVDA5CNFSM4ID3YCVKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4G7K7YOA>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ADQ3VLMXDS4H6R326ECL4NTP7UFVDANCNFSM4ID3YCVA>.

Commit:
Merge branch 'develop' into chore/116-enhance-company-create-preview-styling

medelman17 commented:
@TerrellLinnell: I just merged current develop into this branch for you; do not force push--pull down these changes and incorporate into your local before pushing again.

Commit:
chore: fix certain styling

Commit:
Merge branch 'develop' into chore/116-enhance-company-create-preview-styling

TerrellLinnell committed:
feat: Reform categories and links

Reform categories and links to be reusable for the company and preview sections

#115 #114 #63 #64

TerrellLinnell committed:
Merge pull request #157 from codexstanford/feat/links-and-categories-preview

feat: Reform categories and links

Commit:
Merge branch 'develop' into chore/116-enhance-company-create-preview-styling

TerrellLinnell committed:
fix: Remove one column from large list of categories in preview

Preview overflowed when there was a lot of categories, removed one column so it does not overflow anymore

#115

TerrellLinnell committed:
Merge pull request #158 from codexstanford/fix/categories-in-preview-overflow

fix: Remove one column from large list of categories in preview

Commit:
chore: add null check for links; clear console; tweak now config

Commit:
fix: amplify-related SSR bugs
